ai/prompt/v0.1.0/schema.json

```diff
@@ -28,7 +33,7 @@
           "title": "Order"
         }
       },
-      "required": ["name", "prompt"]
+      "required": ["assistantId", "name", "prompt"]
```

Comment:
Should `order` be required?

Reply:
I don't know how to manage the sorting in the UI yet and not sure what's best between an `undefined` value and a arbitrary value.